content/docs/intro/concepts/how-pulumi-works.md

```diff
@@ -83,15 +83,17 @@ contentBucket, _ := s3.NewBucket(ctx, "content-bucket", nil)
 ```csharp
 using System.Threading.Tasks;
 using Pulumi;
-using Pulumi.Aws;
+using Aws = Pulumi.Aws;
 
 class Program
 {
-    static Task Main() =>
-        Deployment.Run(() => {
-            var mediaBucket = new Aws.S3.Bucket("media-bucket");
-            var contentBucket = new Aws.S3.Bucket("content-bucket");
-        });
+    static Task<int> Main() => Deployment.RunAsync<MyStack>();
+}
+
+public MyStack : Stack
+{
+    var mediaBucket = new Aws.S3.Bucket("media-bucket");
+    var contentBucket = new Aws.S3.Bucket("content-bucket");
 }
```

Comment:
Curious - Is this a "normal" way to express this?

Reply:
That's a bug! Will fix.